lib/cadet_web/controllers/assessments_controller.ex

```diff
@@ -187,7 +187,9 @@ defmodule CadetWeb.AssessmentsController do
               "The library used for this question"
             )
 
-            solution_template(:string, "Solution template for programming questions")
+            solutionTemplate(:string, "Solution template for programming questions")
```

Comment:
Frontend will updated for this right?

Reply:
actually both front and backend are using camel case already. The doc was outdated.